README_PPX.md

```diff
@@ -79,6 +90,20 @@ type bar = (foo [@generic foo_generic], string) result [@@deriving irmin]
 val bar_t = Irmin.Type.(result foo_generic string)
 ```
 
+Built-in abstract types such as `unit` are assumed to be represented in
+`Irmin.Type`. This behaviour can be overridden with the []`[@nobuiltin]`
+attribute:
+
+
+```ocaml
+type unit = string [@@deriving irmin]
+
+type t = unit [@nobuiltin] [@@deriving irmin]
+
+(* generates the value *)
+let t = string_t (* not [Irmin.Type.t] *)
```

Comment:
why `string_` ?

Reply:
Good point, it's actually `[unit_t]` here. This is not a particularly clear example.

Reply:
Potentially fixed by https://github.com/mirage/irmin/pull/994/commits/e3b4d30599f2ad7d91a751d37c370cf19142fafc.